v1/pdf.js

```diff
@@ -22,23 +22,13 @@ module.exports = options => ({
     operations: {
         generatePDF: (hyper, req) => {
             const rp = req.params;
-            const newProbability = options.new_probability || 0;
             return hyper.get(new URI([rp.domain, 'sys', 'page_revisions', 'page', rp.title]))
             .then((latestRevision) => {
-                if (options.new_uri && (Math.random() < newProbability || req.query.new_pdf)) {
-                    const newResult = hyper.get(new URI(
+                if (options.new_uri) {
+                    return hyper.get(new URI(
                         `${options.new_uri}/${rp.domain}/v1/`
                         + `pdf/${encodeURIComponent(rp.title)}/a4/desktop`
-                    ))
-                    .catch((e) => {
```

Comment:
I'd still keep logging

Reply:
Won't throwing the error here make RB log it automatically? We don't want to end up with duplicate log entries.